packages/table/src/table.tsx

```diff
@@ -1635,12 +1669,16 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         const didScrollLeftChange = nextScrollLeft !== viewportRect.left;
 
         if (didScrollTopChange || didScrollLeftChange) {
-            // we need to modify the body element explicitly for the viewport to shift
+            // we need to modify the scroll container explicitly for the viewport to shift. in so
+            // doing, we add the size of the header elements, which are not technically part of the
+            // "grid" concept (the grid only consists of body cells at present).
```

Comment:
This was actually a regression that happened during the frozen rows/columns refactor. But all that broke was auto-scrolling as you use arrow keys to move the focus cell around. Still, works again now. 👍